404.html

```diff
@@ -0,0 +1,20 @@
+---
+---
```

Comment:
What are these?

Reply:
This is called "front matter". https://jekyllrb.com/docs/frontmatter/

It's used by Jekyll to parse files. Even if there are no parameters, I'm pretty sure the top-level html files still need them.

Reply:
Following-up. Question answered (hopefully) :P